pallets/transfer-allowlist/Cargo.toml

```diff
@@ -31,7 +31,10 @@ sp-std = { git = "https://github.com/paritytech/substrate", default-features = f
 xcm = { git = "https://github.com/paritytech/polkadot", default-features = false, branch = "release-v0.9.32" }
 
 [dev-dependencies]
+pallet-authorship = { git = "https://github.com/paritytech/substrate", default-features = false, branch = "polkadot-v0.9.32" }
 pallet-balances = { git = "https://github.com/paritytech/substrate", default-features = true, branch = "polkadot-v0.9.32" }
+pallet-fees = { path = "../fees" }
```

Comment:
You do not need to add `pallet-fees`/`pallet-treasury`/`pallet-authorship` to test this. You can do it much more manageably with the mock in `libs/mocks/src/fees.rs`

Reply:
Ah ok, thanks! Yeah that would be a lot more convenient :)  I wanted to verify the reserve/unreserve amounts, and verify the pallet was using the FeeKey correctly and ensure that the reserve/unreserve logic was all being handled correctly in here.